### PR TITLE
Feature: add cron setting for the ldap settings

### PR DESCRIPTION
**Pull request by markelog**
* Move ldap configuration read to special function

* Introduce cron setting (no docs for it yet, pending approval)

### CODE REVIEW

**Review by torkelo — APPROVED** [review]
LGTM 
@xlson do you have any thoughts?

File: conf/defaults.ini [hunk 1]
```diff
@@ -393,6 +393,7 @@ headers =
 enabled = false
 config_file = /etc/grafana/ldap.toml
 allow_sign_up = true
+sync_cron = @hourly
```
Comment (xlson) [inline conf/defaults.ini]: I'd add a comment explaining that this is related to ldap sync in enterprise.

Something like: `# LDAP backround sync (Enterprise)`

**Review by xlson — APPROVED** [review]
Looks good! Add a comment explaining the setting and merge.

File: conf/defaults.ini (hunk 1 above)
Comment (markelog) [reply conf/defaults.ini]: Oh, I definitely want to do that! Was thinking to add such comment with the rest of the documentation, but I can do it in this pull right away :)

**Pull request merged**